pillar/fluentd/server.sls

```diff
@@ -73,6 +73,18 @@ fluentd:
             - format: json
             - port: 9999
             - keep_time_key: 'true'
+        - directive: source
+          attrs:
+            - '@id': secure_input
+            - '@type': forward
+            - port: 5001
+            - nested_directives:
+              - directive: transport
+                directive_arg: tls
+                attrs:
+                  - cert_path: '/etc/ssl/certs/log-input.crt'
+                  - private_key_path: '/etc/ssl/certs/log-input.key'
```

Comment:
Now we just need to generate the certificates...

Reply:
The certs are already copied over from vault for the nginx config